js/tests/components/SnippetPreviewSection.test.js

```diff
@@ -19,3 +19,65 @@ describe( "SnippetPreviewSection", () => {
 		expect( tree ).toMatchSnapshot();
 	} );
 } );
+
+describe( "mapEditorDataToPreview", () => {
+	it( "Hyphenates a space in the URL.", () => {
```

Comment:
I could be missing something but I think this test does not add anything over the one where you test all the spaces? More tests don't hurt ofcourse, except when we change functionality later, and have to edit more tests than needed ;).